poppy/poppy_core.py

```diff
@@ -2059,7 +2059,7 @@ def __init__(self, name="unnamed optic", transmission=None, opd= None, opdunits=
             elif isinstance(opd, fits.HDUList):
                 # load from fits HDUList
                 self.opd_file='supplied as fits.HDUList object'
-                self.opd = opd[0].data
+                self.opd = opd[0].data.copy()
                 self.opd_header = opd[0].header
```

Comment:
Should we also copy the FITS header, for consistency? I can't think of any practical circumstances where header edits mid calculation are likely to occur, but it seems odd to copy the data array but not the header.